runtime/common/src/lib.rs

```diff
@@ -232,3 +234,50 @@ pub mod constants {
 		items as Balance * 15 * CENTI_CFG + (bytes as Balance) * 6 * CENTI_CFG
 	}
 }
+
+pub mod parachains {
+	pub mod karura {
+		pub const ID: u32 = 2000;
+		pub const KUSD_KEY: &[u8] = &[0, 129];
```

Comment:
What is they key used for?

Reply:
A token is identified as a pair of `(ParaId, GeneralKey)`. The `ID` is karura's Para Id in Kusama and `KUSD_KEY` is the general key value of KSUD. KSUD in the network is identified as `(200, [0, 129])`.